MathLib/LinAlg/BLAS.cpp

```diff
@@ -20,13 +20,29 @@ namespace MathLib { namespace BLAS
 {
 
 // Explicit specialization
+// Computes the Manhattan norm of x
```

Comment:
Just thought that Manhattan is in a line with Euclidean and Maximum, i.e. no numbers included in the name.
BTW: Wikipedia lists Manhattan twice :smile:
> The taxicab metric is also known as rectilinear distance, L1 distance or \ell_1 norm (see Lp space), snake distance, city block distance, Manhattan distance or Manhattan length
> (https://en.wikipedia.org/wiki/Taxicab_geometry)